platform/glfw/glfw_view.cpp

```diff
@@ -381,7 +381,6 @@ void GLFWView::onFramebufferResize(GLFWwindow *window, int width, int height) {
     GLFWView *view = reinterpret_cast<GLFWView *>(glfwGetWindowUserPointer(window));
     view->fbWidth = width;
     view->fbHeight = height;
-    view->bind();
```

Comment:
![screenshot from 2017-03-28 18-40-07](https://cloud.githubusercontent.com/assets/996307/24414075/ff0b5cb6-13e5-11e7-8a92-cdc013cb5c84.png)

This is how it looks when resizing on Linux with this patch. I'm not sure why it behaves differently on both platforms, but if you don't feel like debugging glfw a `#ifdef` would do it.